app/controllers/shelters_controller.rb

```diff
@@ -1,11 +1,11 @@
 class SheltersController < ApplicationController
-  before_action :set_headers
+  before_action :set_headers, except: [:index]
```

Comment:
Just for consistency, could you use a `before_action` for the index headers too?

Reply:
Yep, will do.